crates/e2e/tests/e2e/protocol_fee.rs

```diff
@@ -231,7 +231,7 @@ async fn combined_protocol_fees(web3: Web3) {
         )
         .await
         .unwrap();
-        new_market_order_quote.quote.buy_amount != market_quote_before.quote.buy_amount
+        new_market_order_quote.quote.buy_amount > market_quote_before.quote.buy_amount * 2
```

Comment:
from where is coming the `2`?

Reply:
This `2` isn't really based in anything. The actual new price is orders of magnitude bigger and the difference in buy amount due to gas prices is way smaller than that. I just wanted to make the smallest possible change without having to deal with floating point math.
If you feel like it's helpful or necessary I can add a comment but I wouldn't want to make this a constants since it's not relevant for anything else.

Reply:
Thanks a lot! :pray: please, could you add a comment for it?

Reply:
Done